build/release/release.sh

```diff
@@ -16,6 +16,34 @@ if [[ $1x = x ]]; then
     TAG=$DATE-$VERSION
 else
     TAG=$1
+
+    # If a tag is specified, ensure that that tag is present and checked out.
+    if [[ $TAG != $(git tag -l --points-at HEAD) ]]; then
+        echo "tag $TAG not checked out"
+        exit 1
+    fi
+
+    # Build lnd to extract version.
+    go build github.com/lightningnetwork/lnd/cmd/lnd
```

Comment:
In case lnd isn't checked out in that directory, wouldn't `go build ./cmd/lnd` be better?

Reply:
I've taken this from how the release binaries are built further down in this file. I think the whole script assumes that it is started from the repo root.

Reply:
Ah, I didn't see that. Makes sense to do it the same way here then.